pkg/storage/calico/api_object_versioner.go

```diff
@@ -31,8 +29,8 @@ func (a APIObjectVersioner) ObjectResourceVersion(obj runtime.Object) (uint64, e
 		return 0, nil
 	}
 	if strings.ContainsRune(version, '/') == true {
-		conv := conversion.NewConverter()
-		crdNPRev, k8sNPRev, _ := conv.SplitNetworkPolicyRevision(version)
+		revs := strings.Split(version, "/")
+		crdNPRev, k8sNPRev := revs[0], revs[1]
```

Comment:
Think @caseydavenport was making some changes around this recently. Would like his 👍  on this one.